packages/@aws-cdk/aws-ecs-patterns/test/fargate/integ.queue-processing-fargate-service-public.ts

```diff
@@ -16,7 +16,7 @@ new QueueProcessingFargateService(stack, 'PublicQueueService', {
   assignPublicIp: true,
   healthCheck: {
     command: ['CMD-SHELL', 'curl -f http://localhost/ || exit 1'],
-    interval: Duration.seconds(5),
+    interval: Duration.seconds(6),
```

Comment:
I needed to change this to something greater than 5. Cloudformation was giving this error:

```
aws-ecs-patterns-queue-public | 11:14:45 AM | CREATE_FAILED        | AWS::ECS::TaskDefinition                    | PublicQueueService/QueueProcessingTaskDef (PublicQueueServiceQueueProcessingTaskDef858327F6) Resource handler returned message: "Invalid request provided: Create TaskDefinition: Health check interval must be less than or equal to the maximum allowed value 300 for container 'QueueProcessingContainer' 
```